### PR TITLE
AST 6.1 bump

### PR DESCRIPTION
**Pull request by suShirow**
Have a nothing starwich

### CODE REVIEW

File: src/parser/jobs/ast/index.tsx [hunk 1]
```diff
@@ -29,12 +29,22 @@ export const ASTROLOGIAN = new Meta({
 	</>,
 	supportedPatches: {
 		from: '6.0',
-		to: '6.08',
+		to: '6.1',
 	},
 	contributors: [
-		{user: CONTRIBUTORS.OTOCEPHALY, role: ROLES.DEVELOPER},
+		{ user: CONTRIBUTORS.SUSHIROU, role: ROLES.MAINTAINER }, {user: CONTRIBUTORS.OTOCEPHALY, role: ROLES.DEVELOPER},
```
Comment (ayyaruq) [inline src/parser/jobs/ast/index.tsx]: GH will prob mangle the whitespace but you get the idea:
```suggestion
		{ user: CONTRIBUTORS.SUSHIROU, role: ROLES.MAINTAINER },
		{ user: CONTRIBUTORS.OTOCEPHALY, role: ROLES.DEVELOPER },
```

**Review by ayyaruq — COMMENTED** [review]
Not gonna force it, poke me to merge anyway if yolo since it's just style.

**Review by ayyaruq — APPROVED** [review]
Approved, waiting on core 6.1 support for merge.

**Comment by suShirow** [comment]
I removed my changelog (._.) sori sori...

**Review by ayyaruq — APPROVED** [review]

**Pull request merged**